client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -22,10 +22,10 @@ const ConnectedSingleSiteWpcom = connectOptions( ( props ) => {
 		upsellBanner = (
 			<UpsellNudge
 				className="themes__showcase-banner"
-				event="calypso_themes_list_install_themes"
-				feature={ FEATURE_UPLOAD_THEMES }
+				event="calypso_themes_list_premium_themes"
+				feature={ FEATURE_PREMIUM_THEMES }
 				plan={ PLAN_BUSINESS }
-				title={ translate( 'Upload your own themes with our Business and eCommerce plans!' ) }
+				title={ translate( 'Unlock ALL premium themes with our Business and eCommerce plans!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Upload your own themes with our Premium and Business plans!' )` *ES Score: 8*